packages/editor/src/components/ChooseTab.tsx

```diff
@@ -309,6 +326,21 @@ export default function ChooseTab() {
           </Tooltip>
         </li>
       </ul>
+      <hr />
```

Comment:
This looks good and is working for me. When I used this option I did notice that the Advanced Settings seemed to run really slow. I'm not sure if it was the config I tried uploading. However, compared to the text based format this does seem slower. I think we'll want to weigh pros/cons with speed with text based editor and using this library.

Reply:
I think maybe because it has to also do a full data pull with this option? with the text editor you're changing things after the data has been updated, which i also thought was no ideal UX because you can also change the data there.